drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -126,13 +126,24 @@ def self.options_for_type(pick_list_type, user:, relation_id: nil)
         Hmis::File.all_available_tags.map do |tag|
           {
             code: tag.id,
```

Comment:
This "code" is what should be resolved on the GraphQL File type. Instead the tags field resolves like `["Birth Certificate", "Green Card"]`. That's (at least part of) why the tags dropdown is not populating when you try to re-open and edit a file. @ZachCMP

Reply:
If you need to resolve the names (and/or need to receive the names in the processer), then the "code" here should also be the name.

Reply:
Fixed